Onboarding/Onboarding/Utilities/CommonDriver.cs

```diff
@@ -14,14 +14,16 @@ namespace Onboarding.Utilities
     public class CommonDriver
     {
         public IWebDriver driver;
```

Comment:
so you have done what I said you, I think you don't have to change that to static it should work, try it out!

Reply:
Done with no static in driver